cg/cli/workflow/balsamic/options.py

```diff
@@ -5,6 +5,14 @@
 OPTION_DRY = click.option(
```

Comment:
There is a global constant dry-run option if you would like to switch

Reply:
Sure! [e0b78cc](https://github.com/Clinical-Genomics/cg/pull/1477/commits/e0b78cc888edbb5365d58423fcc73d6a0f92cf5a)